substrate/primitives/runtime/Cargo.toml

```
@@ -14,7 +14,7 @@ readme = "README.md"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-tuplex = "0.1.1"
+tuplex = { git = "https://github.com/gavofyork/tuplex.git", default-features = false }
```

```suggestion
tuplex = "0.1.2"
```

https://github.com/oooutlk/tuplex/pull/1

---

Nice!